src/commands/bootstrap-commands.ts

```diff
@@ -56,9 +56,11 @@ export const bootstrapCommands = ({
       if (goodbyeChannel.type !== 'text') {
         console.error('log ');
       }
-      (goodbyeChannel as TextChannel).send(
-        `${member.user} (**${member.user?.username}**) has left us! :(`
-      );
+      if (member.roles.cache.has('720740563472744521')) {
```

Comment:
shouldn't this be checking against the `SUSPENDED_ROLE` [environment variable](https://github.com/bradtaniguchi/discord-bot-test/blob/master/src/config/get-config.ts#L58)?

I dislike "magic strings/numbers"

Reply:
Yeh that's a better approach. I'll refactor it :)